### ISSUE TITLE
Phase 8 | Deep Learning

### ISSUE THREAD

**Issue by kirstenlynn-ops** [kirstenlynn-ops/Portfolio#23]
Tasks for Phase 8:

- [ ] Prepare features for neural network
- [ ] Build Keras model architecture
- [ ] Train + validate model
- [ ] Plot training loss/accuracy
- [ ] Compare against Spark model
